### PR TITLE
Added SQLite and simple example commands

### PR DESCRIPTION
**Pull request by Zabuzard**
Since the bot will be using a database soon anyways, I decided to add SQLite as dependency and also a simple command to showcase how it can be used (in order to get started).

The added commands are:

- `!dbput <key> <value>`
- `!dbget <value>`

and it acts basically as a simple persisted `Map<String, String>`.

Command usage example

### CODE REVIEW

**Review by Tais993 — APPROVED** [review]

**Review by marko-radosavljevic — DISMISSED** [review]

**Comment by marko-radosavljevic** [comment]
Maybe add some tests to this before merging, if you think it will stay.

**Comment by d3vel0per** [comment]
I was holding off for the command structure story. I think Zabu's intention here was to just set an example and not for the long term. The last discussion I had on Discord was both Istannen and Tijs are against text commands. Probably slash commands are the way to go.

**Comment by marko-radosavljevic** [comment]
Yeah, I'm just reminding for every case. It's easy to forget about testing, and then it's too late. ^^

It's also something we didn't discuss, how much testing we want, etc. Makes sense that only tested things are merged into develop.

**Pull request merged**
